projectional/src/main/java/jetbrains/jetpad/projectional/cell/ProjectionalRoleSynchronizer.java

```diff
@@ -34,6 +34,7 @@
   <ContentT> void supportListContentKind(ContentKind<ContentT> kind, Function<ContentT, List<SourceT>> fromContent);
   void setOnLastItemDeleted(Runnable action);
   void setPlaceholderText(String text);
+  void disablePlaceholder(Runnable onLastItemDeleted);
```

Comment:
There is no need to pass onLastItemDeleted. onLastItemDeleted should handle the disabled placeholder case.